mappings/net/minecraft/item/FoodItemSetting.mapping

```diff
@@ -0,0 +1,37 @@
+CLASS awp net/minecraft/item/FoodItemSetting
+	CLASS awp$a Builder
+		FIELD a hunger I
+		FIELD b saturation F
+		FIELD c wolfFood Z
+		FIELD d alwaysEdible Z
+		FIELD e eatenFast Z
+		FIELD f statusEffectChances Ljava/util/List;
+		METHOD a wolfFood ()Lawp$a;
+		METHOD a setSaturation (F)Lawp$a;
+			ARG 1 amount
+		METHOD a setHunger (I)Lawp$a;
```

Comment:
Would renaming this to `hunger` and `saturation` look better?

Reply:
I think it would be better as it would fit how methods are already named in the  existing BlockSettings and ItemSettings builders e.g. new Item.Settings().durability(500).build()